examples/gnrc_networking/README.md

```diff
@@ -18,13 +20,15 @@ First, create a tap interface:
 Now you can start the `gnrc_networking` example by invoking `make term`. This should automatically connect to the `tap0` interface. If
 this doesn't work for some reason, run `make` without any arguments, and then run the binary manually like so (assuming you are in the `examples/gnrc_networking` directory):
 
+    sudo bin/native/gnrc_networking.elf tap0
```

Comment:
Ha, see, I didn't know that `PORT=tap1 make term` was a thing :D I'll just add that then? I'm not particularly invested in this entire paragraph, I just saw that there was a line missing and I tried to fill it out ;)